lib/action_policy/policy/aliases.rb

```diff
@@ -58,6 +64,10 @@ def rules_aliases
               {}
             end
         end
+
+        def method_added(name)
+          rules_aliases.delete(name) if public_method_defined?(name)
```

Comment:
Why do we need to check for `public_method_defined?`?

Reply:
See my other comment: https://github.com/palkan/action_policy/pull/22#issuecomment-401236630

I figured it's safer than just deleting aliases when adding private methods. I'm not sure if `public_send` goes on to the superclass to check for a public method if a public method doesn't exist on the subclass, or if it just raises?

Reply:
>goes on to the superclass to check for a public method if a public method doesn't exist on the subclass, or if it just raises?

It raises if a private method is defined with the same name; otherwise it looks up in superclass.

Let's live the `public_method_defined?` then.

We need to add a note to the docs that only public method are considered as rules, and aliases do not affect private/protected methods.

Reply:
Ok, that's done now :)